lib/middleman-prismic/preview.rb

```diff
@@ -0,0 +1,22 @@
+module Middleman
+  module Prismic
+    class Preview
+      def initialize(app, options={})
+        @app = app
+      end
+
+      def call(env)
+        req = ::Rack::Request.new(env)
+        if req.path =~ %r(^/preview)
+          token = Shellwords.escape(req.params["token"])
+
+          Kernel.system "middleman prismic --ref #{token}"
```

Comment:
Is this a code injection vulnerability?

Reply:
We're using `Shellwords.escape` above, so we should be ok?

Reply:
Ah, I wasn't sure how safe `Shellwords.escape` was 👍